b-lab-org/ember-cli-impact-core
# [Wellness] Remove Materialize icon font

## Overview
When the new impact-app dashboard was introduced, we also introduced loading the Google Material Icons font. This left us with two fonts performing the same function.

In addition, the Google font was loaded as a synchronous request, which incurs a performance cost.

[story](https://www.pivotaltracker.com/story/show/157573296)

## What is new
- [ember-material-icons](https://github.com/ynnoj/ember-material-icons) which brings in a `material-icon` component and includes the Google font in our build
- new component, `material-icon-tooltip` replaces `impact-tooltipped-icon` (has a similar API)

The old way of using material icons (assuming they existed in the Materialize font):
`<i class="mdi-something-something"></i>`

The new way:
`{{material-icon "icon_name"}}`

## How to test
- A visual regression would be great, making sure all of our icons still appear as expected
- Icons still toggle in instances where they should toggle
- Icons still have tooltips where the used to
